server/src/api/socket/initiative/__init__.py

```diff
@@ -349,17 +356,22 @@ async def change_initiative_order(sid: str, raw_data: Any):
 
 @sio.on("Initiative.Turn.Update", namespace=GAME_NS)
 @auth.login_required(app, sio, "game")
-async def update_initiative_turn(sid: str, turn: int):
+async def update_initiative_turn(sid: str, raw_data: Any):
+    data = InitiativeTurnUpdate(**raw_data)
     pr = game_state.get(sid)
 
     location_data: Initiative = Initiative.get(location=pr.active_location)
     json_data = json.loads(location_data.data)
+    data_len = len(json_data)
```

Comment:
`data_len` is a correct name, but maybe `number_of_turns`/`total_turns`/`turn_count` or something similar is a clearer name for the actual use of the variable.